src/side-navigation/parts.tsx

```diff
@@ -293,6 +293,7 @@ function Section({ definition, activeHref, fireFollow, fireChange, variant }: Se
         variant === 'section-group' && styles['section--no-ident'],
         isVisualRefresh && styles.refresh
       )}
+      disablePaddings={true}
```

Comment:
these were previously overridden via custom CSS (`className` above`), explicitly disabling them is better for future predictability